BSPConvertLib/Source/EntityConverter.cs

```diff
@@ -845,6 +879,8 @@ private string GetMomentumAmmoName(string q3AmmoName)
 		private void ConvertItem(Entity itemEnt)
 		{
 			itemEnt["resettime"] = GetItemRespawnTime(itemEnt);
+			if (string.IsNullOrEmpty(itemEnt["count"]) || itemEnt["count"] == "0")
```

Comment:
This can be improved slightly:
```
if (!itemEnt.TryGetValue("count", out var count) || count == "0")
    count = "30";
```

Then use count on lines 885 and 887.